src/app/autocomplete-input/autocomplete-input.component.ts

```diff
@@ -62,21 +80,22 @@ export class AutocompleteInputComponent {
         this.schemaKeysStoreService.forPath(state.path).filter(item => item.startsWith(state.query))
       );
     });
-    this.valueChange$
-      .debounceTime(500)
-      .subscribe(value => {
-        this.valueChange.emit(value);
-      });
   }
 
-  modelChange(value: string) {
-    this.value = value;
-    this.valueChange$.next(this.value);
+  ngOnInit() {
+    if (this.tagsEnabled) {
+      this.mousetrap = Mousetrap(this.el.nativeElement);
+      this.mousetrap.bind(this.tagShortcut, (event) => {
```

Comment:
Mousestrap is not necessary here and it might be even heavy, you could just `(keyup.space)="onSpaceKeyup($event)"`

Reply:
I am gonna try it. I was thinking to use it if we want a customized shortcut combo.